### PR TITLE
Make Themeing related CSS vars UI Configurable

### PR DESCRIPTION
**Pull request by merttumer**
Change-Id: If6e4096dc37573ff116f218ceb4ca14135820ff7
Signed-off-by: mert <mert.tumer@collabora.com>

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: loleaflet/html/loleaflet.html.m4 [hunk 1]
```diff
@@ -330,5 +330,5 @@ m4_ifelse(MOBILEAPP,[true],
   [    <script src="%SERVICE_ROOT%/loleaflet/%VERSION%/bundle.js" defer></script>
   ])
 )m4_dnl
-    <!--%BRANDING_JS%--> <!-- logo onclick handler -->
+    <!--%BRANDING_JS%--> <!-- logo onclick handler --> <!--%CSS_VARS%-->
```
Comment (kendy) [inline loleaflet/html/loleaflet.html.m4:333]: Let's put the CSS_VARS on a new line :-)

File: wsd/FileServerUtil.cpp [hunk 2]
```diff
@@ -99,4 +99,30 @@ std::string FileServerRequestHandler::uiDefaultsToJSON(const std::string& uiDefa
...
+        return previousVars;
+
+    std::ostringstream styleOSS;
+    styleOSS << "<style>:root {";
+    StringVector tokens(Util::tokenize(cssVars, ';'));
+    for (const auto& token : tokens)
+    {
+        StringVector keyValue(Util::tokenize(tokens.getParam(token), '='));
+
+        styleOSS << keyValue[0] << ":" << keyValue[1] << ";";
```
Comment (kendy) [inline wsd/FileServerUtil.cpp:118]: We should skip the tokens that have no '=', and warn about them to the log...

Comment (kendy) [reply wsd/FileServerUtil.cpp:118]: [oh and similarly if there are more than 1 '=''s :-)

**Review by kendy — COMMENTED** [review]
Looks great in general, just small nitpicks - can you update please?

File: loleaflet/html/loleaflet.html.m4 (hunk 1 above)
Comment (merttumer) [reply loleaflet/html/loleaflet.html.m4]: done

**Review by kendy — APPROVED** [review]
Beautiful, I love it - thank you! :-)

**Comment by merttumer** [comment]
found a bug looking that

**Pull request merged**